Gemfile

```diff
@@ -25,7 +25,8 @@ end
 
 gem "puppet", :path => File.dirname(__FILE__), :require => false
 gem "facter", *location_for(ENV['FACTER_LOCATION'] || ['> 1.6', '< 3'])
-gem "hiera", *location_for(ENV['HIERA_LOCATION'] || '~> 1.0')
+# gem "hiera", *location_for(ENV['HIERA_LOCATION'] || '~> 1.0')
+gem "hiera", :path => "#{File.dirname(File.dirname(__FILE__))}/hiera", :require => false
```

Comment:
Why not leave it the way it was (where HIERA_LOCATION could override a gem), but expand the gem range to be hiera 1.x or 2.x?